src/main/riddl/Venues/Locations/locationMessages.riddl

```diff
@@ -0,0 +1,58 @@
+command EstablishLocation is {
+  locationId: LocationId briefly "A unique identifier for this Location. This must be provided by the calling client.",
+  name: String,
+  description: String,
+  directions: String,
+  latLong: LatLong,
+  url: URL?,
+  capacity: Integer?,
+  accessible: Boolean,
+  amenities: String*,
+  entrances: Address+,
+  reservations: ReservationId*
```

Comment:
I think you should model reservations for a Location as a projection. Reservations by Location is important to query, but is not defining for a Location.